packages/poc-v2/contracts/StateUtils.sol

```diff
@@ -332,4 +332,11 @@ contract StateUtils {
         updateUserState(userAddress, fromBlock);
         return getValueAt(users[userAddress].shares, fromBlock);
     }
+
+    // TODO: delete this function after testing has completed
+    function testUpdateCurrentApr(uint _totalStaked) public {
+        totalStaked.push(Checkpoint(block.number, _totalStaked));
+        updateCurrentApr();
+        totalStaked.pop();
+    }
```

Comment:
I really like this idea, but note that ultimately we have to test without convenience functions